### PR TITLE
feat: customizable major and minor wording to bump new versions

### LINKED ISSUE #32: Configurable Major / Minor / Patch phrases
- I'd like a way to define new phrases to the ones which trigger different bump types
- For example there's a repo where the phrase 'Add' or 'Adds' should result in a minor bump

### PR DESCRIPTION
**Pull request by Eomm**
Fix #32

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -25,11 +25,12 @@ Toolkit.run(async tools => {
     return
   }
 
+  const majorWords = process.env['INPUT_MAJOR-WORDING'].split(',')
```
Comment (phips28) [inline index.js:28]: `process.env['INPUT_MAJOR-WORDING'] ? process.env['INPUT_MAJOR-WORDING'].split(',') : ['BREAKING CHANGE', 'major']`

File: index.js [hunk 2]
```diff
@@ -25,11 +25,12 @@ Toolkit.run(async tools => {
...
+  const minorWords = process.env['INPUT_MINOR-WORDING'].split(',')
```
Comment (phips28) [inline index.js:29]: `process.env['INPUT_MINOR-WORDING'] ? process.env['INPUT_MINOR-WORDING'].split(',') : ['feat', 'minor']`

**Review by phips28 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (Eomm) [reply index.js:28]: I have added these values as default in the `action.yml`

Should I move them?

Comment (phips28) [reply index.js:28]: ahh ok. than its fine

**Review by phips28 — APPROVED** [review]

**Comment by phips28** [comment]
@Eomm there is a conflict.

**Comment by kubikowski** [comment]
Is there a reason we decided to represent the `majorWords` and `minorWords` as strings in yaml?
I would imagine representing them as lists would be the preferred syntax. For example:
```
major-wording:
  default:
    - BREAKING_CHANGE
    - major
```

**Comment by Eomm** [comment]
> Is there a reason we decided to represent the `majorWords` and `minorWords` as comma delimited strings in yaml?
> I would imagine representing them as lists would be the preferred syntax. For example:
> 
> ```
> major-wording:
>   default:
>     - BREAKING_CHANGE
>     - major
> ```

Because you cannot. You must follow the [`action.yml` format spec](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault)

**Pull request merged**